mantle/kola/harness.go

```diff
@@ -683,7 +684,7 @@ func filterDenylistedTests(tests map[string]*register.Test) (map[string]*registe
 // register tests in their init() function.  outputDir is where various test
 // logs and data will be written for analysis after the test run. If it already
 // exists it will be erased!
-func runProvidedTests(testsBank map[string]*register.Test, patterns []string, multiply int, rerun bool, rerunSuccessTags []string, pltfrm, outputDir string, propagateTestErrors bool) error {
+func runProvidedTests(testsBank map[string]*register.Test, patterns []string, multiply int, rerun bool, rerunSuccessTags []string, pltfrm, outputDir string, propagateTestErrors bool, useExitWarnCode77 bool) error {
```

Comment:
i'm not a big fan of changing all of these function signatures. I think a better way may be to have an error type var and also an option. See what you think about https://github.com/dustymabe/coreos-assembler/commit/fec216316afbaa5081626e0a4f4c05e0cbdb0133 (haven't had a chance to test it yet, but it does compile).

If we end up using that code I would want to incorporate the changes in some of the previous commits in this PR so we don't change the function signatures and then change them back.

Reply:
>See what you think about https://github.com/dustymabe/coreos-assembler/commit/fec216316afbaa5081626e0a4f4c05e0cbdb0133

Hmm, code already passes `kola.Options.NoTestExitError` option thru several wrappers to  `runProvidedTests`'s  `propagateTestErrors` , so i don't it's a good idea to mix direct and indirect usage of `kola.Options.*`

Reply:
Yep, i'll return back to work on Aug. 21 and will update this PR